bin/pycbc_multi_inspiral

```diff
@@ -194,28 +269,32 @@ with ctx:
     # change if needed. Segments is only used to get tlen etc. which is
     # same for all ifos, so just send the first ifo
     template_mem = zeros(tlen, dtype=complex64)
-    
+
     #Read the sky grid or the single sky position
     if args.sky_grid is not None and args.ra is not None and args.dec is not None:
         parser.error('Give either a sky grid or a sky position, not both')
-    
+
     if args.sky_grid is not None:
         sky_grid = h5py.File(args.sky_grid, 'r')
         ra = np.array(sky_grid['ra'])
         dec = np.array(sky_grid['dec'])
     if args.ra is not None and args.dec is not None:
         ra = np.array([args.ra])
         dec = np.array([args.dec])
-    
+
     sky_positions = np.array([ra, dec])
     num_sky_positions = sky_positions.shape[1]
     positions_array = np.arange(num_sky_positions)
 
-    # Calculate time delays to each detector for each sky position and apply time slide shifts
+    # Calculate time delays to each detector and store time slide shifts
+    # in a dedicated dictionary
     slide_ids = np.arange(1 + args.num_slides)
     time_slides = {
         ifo: args.slide_shift * slide_ids * n_ifo
         for n_ifo, ifo in enumerate(args.instruments)}
+    # Given the time delays in time_slides, the following dictionary returns the
+    # time delay indices evaluated wrt the geocenter:
+    # (time delay from geocenter + timeslide)*sampling_rate
```

Comment:
```suggestion
    # Given the time delays in time_slides, the following dictionary keeps the
    # time delays evaluated wrt the geocenter, in units of samples, i.e.
    # (time delay from geocenter + timeslide)*sampling_rate
```

Reply:
Part of my suggestion has been addressed, but please do include the rest as well - I think the comment becomes much clearer.